### PR TITLE
Added Instapaper Bee

### PR DESCRIPTION
**Pull request by adampetrovic**
Add a url from an event to Instapaper

### CODE REVIEW

File: bees/instapaperbee/instapaperbee.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+)
+
+type InstapaperBee struct {
+	bees.Bee
+
+	username string
+	password string
+}
+
+func (mod *InstapaperBee) Run(cin chan bees.Event) {
```
Comment (muesli) [inline bees/instapaperbee/instapaperbee.go]: I think you can omit this function, as that's the standard implementation embedded by `bees.Bee`.

File: bees/instapaperbee/instapaperbeefactory.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+			Name:        "password",
+			Description: "Instapaper Password",
+			Type:        "string",
+			Mandatory:   true,
+		},
+	}
+	return opts
+}
+
+func (factory *InstapaperBeeFactory) Events() []bees.EventDescriptor {
```
Comment (muesli) [inline bees/instapaperbee/instapaperbeefactory.go]: Should be able to omit this function as well.

**Comment by muesli** [comment]
Nice work and another cool addition to Beehive! Just two nitpicky comments I've added.

**Comment by adampetrovic** [comment]
Done!

**Comment by muesli** [comment]
Thank you! :heart:

**Pull request merged**